src/sonic-yang-models/yang-templates/sonic-acl.yang.j2

```diff
@@ -144,7 +152,7 @@ module sonic-acl {
 
 				leaf ETHER_TYPE {
 					type string {
-						pattern "(0x88CC|0x8100|0x8915|0x0806|0x0800|0x86DD|0x8847)";
+					    pattern "0x[6-9a-fA-F][0-9a-fA-F]{2,3}|0x[1-9a-fA-F][0-9a-fA-F]{3,4}";
```

Comment:
>pattern [](http://example.com/codeflow?start=9&length=7)

This change is not related to your PR title or description. And seems the old code is better.
Could you focus on MAC part as original commit?

Reply:
This was discussed earlier also. Please see https://github.com/Azure/sonic-buildimage/pull/7917#discussion_r656718927. User should be able to specify any valid Ethertype for L2 ACL.

Reply:
Is 0x10000 acceptable?
Should we define a shared type?

Reply:
Fixed regex.  This is used only once. Can be moved in future to a shared type if its used/repeated multiple times.

Reply:
0x0800 was acceptable, but not now.